module/TaskManagement/src/TaskManagement/Service/NotificationService.php

```diff
@@ -0,0 +1,51 @@
+<?php 
+
+namespace TaskManagement\Service;
+
+use Zend\View\Renderer\PhpRenderer;
+use Zend\View\Model\ViewModel;
+use Zend\View\Resolver\TemplateMapResolver;
+use Zend\View\Renderer\Zend\View\Renderer;
+
+class NotificationService{
+	
+	
+	/**
+	 *
+	 * @var array of email template paths
+	 */
+	private $emailTemplates;
+	
+	public function __construct($emailTemplates){
+		//TODO: come controllo che sia effettivamente corretto $emailTemplates?
+		$this->emailTemplates = $emailTemplates;
+	}
+	
+	
+	public function sendEmailNotificationForShareAssignment($params){
```

Comment:
Ma... se abbiamo detto che il *reminder* si chiama *Assignment Of Shares* (lo abbiamo deciso assieme e scritto qui su GitHub per ricordarlo) perché usi NotificationForShareAssignment??